Swiftfin/Views/VideoPlayer/Overlays/ChapterOverlay.swift

```diff
@@ -65,8 +65,7 @@ extension VideoPlayer.Overlay {
                             .foregroundColor(accentColor)
                     }
                 }
-                .padding(.leading, safeAreaInsets.leading)
```

Comment:
This padding rework isn't right because it breaks iOS. The entire view is padded so the linear gradient is as well and it makes the buttons when outside of the view not usable (as expected). This padding is somewhat considerable nowadays and we don't want that.

![Simulator Screenshot - iPhone 15 Pro - 2023-09-22 at 22 26 31](https://github.com/jellyfin/Swiftfin/assets/20747774/0bf50adf-24f8-46c0-a869-eaaff0f177cb)

What I am fine with is a check whether it's an iPad and adding more padding. In fact, I do that a lot if you search `.if(UIDevice.isIPad)`.

Reply:
Who watches on aspect fill though :P

Thanks for noticing. These padding changes have been reverted except adding default padding on iPad:

<img width="925" alt="image" src="https://github.com/jellyfin/Swiftfin/assets/22553678/44463294-89a1-4092-8edf-57e6b2ab67bb">
Maybe could use a little bit more padding, but eh, it doesn't look like the text might be cut off anymore which is what I originally wanted to solve so I'm happy with it.

And here's a screenshot of chapter menu open with aspect fill on iPhone with these changes:
<img width="872" alt="image" src="https://github.com/jellyfin/Swiftfin/assets/22553678/1b3722ba-9e45-4660-9757-bd3166d42f4f">